sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingClient.java

```diff
@@ -44,11 +44,21 @@
      *
      * @param formTrainingAsyncClient The {@link FormRecognizerAsyncClient} that the client routes its request through.
      */
-    // TODO (savaity): Still deciding the best approach here, to be redone in #10909
-    public FormTrainingClient(FormTrainingAsyncClient formTrainingAsyncClient) {
+    FormTrainingClient(FormTrainingAsyncClient formTrainingAsyncClient) {
         this.client = formTrainingAsyncClient;
     }
 
+    /**
+     * Creates a new {@link FormRecognizerClient} object. The new {@code FormTrainingClient}
+     * uses the same request policy pipeline as the {@code FormTrainingClient}.
+     *
+     * @return A new {@code FormRecognizerClient} object.
+     */
+    public FormRecognizerClient getFormRecognizerClient() {
```

Comment:
Is this API necessary? The user can just build using the `FormRecognizerClientBuilder` directly.

Reply:
Yeah, we are trying to see the discoverability/usability of the FormRecognizer client via the training client. Moreso, don't want users to use another builder to but just the get the `FormRecognizerClient` using the current builder configurations.